### DISCUSSION TITLE
How to add drill-down/drill-up functionality

### DISCUSSION THREAD

**Discussion by Kentahayakawa** [vega/altair#2613] [Q&A]
Hello all,

I wanted to inquire if there is any drill-down or drill-up functionality for charts in altair-viz currently. If not, are there any future plans to implement this?

Thank you.

**Comment by joelostblom** [comment]
Do you mean something like [sunburst charts in plotly](https://plotly.com/python/sunburst-charts/)? This is generally not possible in Altair, but could be raised as a feature request on the [VegaLite issue tracker ](https://github.com/vega/vega-lite/issues) which is the library Altair is built upon.

**Reply by Kentahayakawa** [reply]
Hi, I was thinking more of drill-up, drill-down in BI tools, such as Tableau and PowerBI, where you can interact with the visualization and "drill-down" to a lower-level category of the same data (for example, from weeks to days)

**Reply by joelostblom** [reply]
Hmm isn't that similar to what a sunburst chart shows? Or could you link to an example? I believe it will still not be possible but just want to make sure I see an example of exactly what you mean first.
